qt/aqt/editor.py

```diff
@@ -1460,4 +1486,14 @@ def set_cloze_button(editor: Editor) -> None:
     )
 
 
+def set_image_occlusion_button(editor: Editor) -> None:
+    action = "show" if editor.note.note_type()["originalStockKind"] == 6 else "hide"
```

Comment:
This should use the enum name, not a number. pylib should export the symbol, and then aqt can reference it via StockNotetypes.ORIGINAL_*

```diff
diff --git a/pylib/anki/models.py b/pylib/anki/models.py
index 7fe27dba3..fe9c9b5f2 100644
--- a/pylib/anki/models.py
+++ b/pylib/anki/models.py
@@ -27,6 +27,7 @@ NotetypeNameIdUseCount = notetypes_pb2.NotetypeNameIdUseCount
 NotetypeNames = notetypes_pb2.NotetypeNames
 ChangeNotetypeInfo = notetypes_pb2.ChangeNotetypeInfo
 ChangeNotetypeRequest = notetypes_pb2.ChangeNotetypeRequest
+StockNotetype = notetypes_pb2.StockNotetype

 # legacy types
 NotetypeDict = dict[str, Any]
```